src/main/java/org/mifos/connector/gsmastub/api/TransactionsApiController.java

```diff
@@ -1,8 +1,8 @@
 package org.mifos.connector.gsmastub.api;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.v3.oas.annotations.*;
```

Comment:
Do not use wildcard for single import.

Reply:
@avikganguly01 and @fynmanoj It is swagger generated code so for that wildcard is used in this case!